### PR TITLE
Add Digital World International (DWI) [Retail]

### PR DESCRIPTION
**Pull request by Jawshy**
#### Changes:
- Added: [Digital World International](http://www.dwidigitalcameras.com.au/) under retail. 
  - No TFA. No Twitter handle. Used their [Facebook page](https://www.facebook.com/DWIAUS/) instead. 

Also referred to as **DWI Digital Cameras**. 

Logo used is less than ideal.

### CODE REVIEW

**Comment by Carlgo11** [comment]
Maybe a

```
git pull upstream
git reset --hard upstream/master
git cherry-pick adcdebc2ac3e8be9718da68a912335489603b008
git push -f
```

?

File: _data/retail.yml [hunk 1]
```diff
@@ -168,4 +174,4 @@ websites:
       url: http://www.zappos.com/
       twitter: Zappos
       img: zappos.png
-      tfa: No
+      tfa: No
```
Comment (Carlgo11) [inline _data/retail.yml]: All the YAMLS should have a newline at the end of the file.

**Comment by Jawshy** [comment]
When trying the above to resolve conflicts with https://github.com/2factorauth/twofactorauth/pull/1946 I got: 

```
G:\Documents\GitHub\twofactorauth [add-computer-alliance = +0 ~1 -0 !]> git pull
 upstream
fatal: 'upstream' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
G:\Documents\GitHub\twofactorauth [add-computer-alliance = +0 ~1 -0 !]>
```

File: _data/retail.yml (hunk 1 above)
Comment (Jawshy) [inline _data/retail.yml]: Fixed.

**Comment by Carlgo11** [comment]
Okay do:

```
git remote add upstream https://github.com/2factorauth/twofactorauth.git
git pull upstream
git reset --hard upstream/master
git cherry-pick adcdebc2ac3e8be9718da68a912335489603b008
git push -f
```

That should work

**Comment by Jawshy** [comment]
Sorted. That was way more painful than it should've been.

**Pull request merged**